doc/api/tty.md

```diff
@@ -123,4 +123,5 @@ added: v0.5.8
 * `fd` {number} A numeric file descriptor
 
 The `tty.isatty()` method returns `true` if the given `fd` is associated with
-a TTY and `false` if is not.
+a TTY and `false` if it is not, including whenever `fd` is not a non-negative
```

Comment:
Seems like there ought to be clearer wording than `not a non-negative integer`. How about just `a negative integer`?

Reply:
I'd also be fine with simply removing the additional text here, as a negative number is not a valid `fd`. But since a negative integer can be returned from a sys call to retrieve a file descriptor if there is an error, I'm OK with leaving it too for explicitness.